sdk/storage/azure-storage-blob/setup.py

```diff
@@ -93,11 +93,13 @@
         'tests.common'
     ]),
     install_requires=[
-        #'azure-core~=0.0.1',
-        'azure-common~=1.1',
+        "azure-core>=1.0.0b1,<2.0.0",
```

Comment:
I believe the dependency verifier in the CI system wants the order of version constraints to be swapped in order to be happy right now. Please double check with the keyvault libraries...

Reply:
Looks like the analysis is passing - but will use the keyvault string anyway